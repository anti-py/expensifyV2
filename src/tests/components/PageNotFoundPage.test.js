import React from 'react';
import { shallow } from "enzyme";
import PageNotFoundPage from "../../components/PageNotFoundPage";

test('should render PageNotFoundPage', () => {
    const wrapper = shallow(<PageNotFoundPage />);
    expect(wrapper).toMatchSnapshot();
})