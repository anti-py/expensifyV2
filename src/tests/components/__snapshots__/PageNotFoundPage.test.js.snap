// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageNotFoundPage 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go -&gt; Home
  </Link>
</div>
`;
